### ISSUE TITLE
How to modify Output Layer in Pretrained Models such as Dl4JResNet50

### ISSUE THREAD

**Issue by trainingemad** [Waikato/wekaDeeplearning4j#63]
### Description
I want to know how can we amend Output Layer methods such as activation function, loss function for pre-trained models such as Dl4JResNet50 using Java API.
### Additional Information

- Weka version:
- wekaDeeplearning4j package version:
- Operating System:
